src/routes/[id]/+page.svelte

```diff
@@ -193,7 +193,7 @@
 					</div>
 				{/if}
 
-				{#each session.messages as message}
+				{#each session.messages as message, i (session.id + i)}
```

Comment:
Need to set a unique key for every instance of `<Article>` so it get's mounted/dismounted every time the session changes.